src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -739,13 +777,37 @@ public void TestPlus()
                 Helper.AssertTemplateResult("2", "{{ 1 | plus:1 }}");
                 Helper.AssertTemplateResult("5.5", "{{ 2  | plus:3.5 }}");
                 Helper.AssertTemplateResult("5.5", "{{ 3.5 | plus:2 }}");
-                Helper.AssertTemplateResult("11", "{{ '1' | plus:'1' }}");

```

Comment:
Perhaps just change this test to

`Helper.AssertTemplateResult("2", "{{ '1' | plus:'1' }}");` to assert the new behavior?